Lib/fontmake/errors.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, msg, source_file):
 
     def __str__(self):
         trail = " -> ".join(
-            f"'{str(_try_relative_path(s))}'"
+            "{!r}".format(f"{str(_try_relative_path(s))}")
```

Comment:
same as above.. Did the linter suggest you to apply this change? If so, surprised it didn't suggest to keep f-strings (which are more efficient)